azure_functions_worker/dispatcher.py

```diff
@@ -217,6 +217,8 @@ def _serialize_exception(self, exc):
         capabilities = dict()
         capabilities[constants.RAW_HTTP_BODY_BYTES] = "true"
```

Comment:
You should be able to remove the `RAW_HTTP_BODY_BYTES` capability if you are using the new capability `RPC_HTTP_BODY_ONLY` I believe. They effectively do the same thing from my understanding except that `rawBody` is not longer needed. \cc @yojagad to confirm if this is the case.